pkgs/development/python-modules/django/4.nix

```diff
@@ -39,23 +39,26 @@
 
 buildPythonPackage rec {
   pname = "Django";
-  version = "4.0.7";
+  version = "4.1";
   format = "pyproject";
 
   disabled = pythonOlder "3.8";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-nG1a02vnmOVi3cqmsXscP/LTxPUppHQytp+5ow+EdGE=";
+    hash = "sha256-Ay+Kb8fPBczRIU5KLiHfzWojudV1xlc8rMjGeCjb5kI=";
   };
 
-  patches = lib.optional withGdal
+  patches = [
+    (lib.optional withGdal
     (substituteAll {
       src = ./django_4_set_geos_gdal_lib.patch;
       geos = geos;
       gdal = gdal;
       extension = stdenv.hostPlatform.extensions.sharedLibrary;
-    });
+    }))
+     ./django_4_sqlite.patch
```

Comment:
Is this a modified patch or can we use fetchpatch instead?

Reply:
it is a modified patch and excludes a diff to a release document (which is not found in the package). I haven't worked with `excludes` yet. Would that be a use case?

Reply:
Excluding changes to files that are part of the patch is exactly the use case .

Reply:
awesome. Thanks for the pointer and applied the change